pkg/kv/kvclient/kvcoord/txn_coord_sender.go

```diff
@@ -1511,6 +1523,16 @@ func (tc *TxnCoordSender) HasPerformedWrites() bool {
 	return tc.hasPerformedWritesLocked()
 }
 
+func (tc *TxnCoordSender) TestingRandomRetryableErrorsEnabled() bool {
+	if tc.testingKnobs.EnableRandomTransactionRetryErrors {
+		return true
+	}
+	if forceTxnRetries && buildutil.CrdbTestBuild {
```

Comment:
`&& buildutil.CrdbTestBuild` requires an extra step when building to debug with goland to make this work. Is this to prevent it from being used in production?

Reply:
Never mind, not using this anyways.